src/core/manager.c

```diff
@@ -3409,18 +3407,15 @@ void manager_set_watchdog(Manager *m, WatchdogType t, usec_t timeout) {
         if (t == WATCHDOG_RUNTIME)
                 if (!timestamp_is_set(m->watchdog_overridden[WATCHDOG_RUNTIME])) {
                         if (timestamp_is_set(timeout))
-                                r = watchdog_set_timeout(&timeout);
+                                watchdog_set_timeout(&timeout);
```

Comment:
given that we actively ignore the return code, please cast result explicitly to `(void)`